### PR TITLE
This Week in Information Security - Week of June 15th

### PR DESCRIPTION
**Pull request by cneill**
__Bio:__
Charles Neill is a Security Engineer at Rackspace, who tests Rackspace products as well as OpenStack projects like Barbican. He is also a developer on [DefectDojo](https://github.com/rackerlabs/django-DefectDojo), an open source defect tracking system using the Django framework.

__Social Media:__
Twitter: [@ccneill](https://twitter.com/ccneill)
Github: [cneill](https://github.com/cneill)
Website: [inventropy.us/blog/](https://inventropy.us/blog/)

__Editing Preference:__
Feel free to edit as needed.

__Posting Preference:__
Desired date to publish: June 15th, 2015

Let me know if you need anything else from me.

Thanks!

### CODE REVIEW

**Comment by ktbartholomew** [comment]
@catlook not sure if you're subscribed to this repo. Would you mind reviewing Charles' blog post?

File: _posts/2015-06-15-this-week-in-infosec-jun-15.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+title: "This Week in Information Security (Week of June 15th)"
+date: 2015-06-15 23:59
+comments: true
+author: Charles Neill
+published: true
+categories:
+    - Security
+---
+
+Welcome back to This Week in Information Security! Sorry if you missed us last week, but posts should follow the schedule you're used to going forward. This week, we have news of two high-profile compromises, a few hair-raising hardware/firmware vulnerabilities, tools from DARPA for searching the "deep" and "dark" web, an opinion piece about vulnerability embargoes in open source software, and more. Finally, we wrap up the week with a fun, interactive article from Bloomberg about the meaning of code, and the people and culture that produce it.
```
Comment (catlook) [inline _posts/2015-06-15-this-week-in-infosec-jun-15.md]: .... code and the people ...

File: _posts/2015-06-15-this-week-in-infosec-jun-15.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+As always, you can find me on Twitter [@ccneill][twitter] if you have any thoughts on this post.
+
+<!-- more -->
+
+# News / Opinions
+
+- [__FBI official: Companies should help us 'prevent encryption above all else'__][fbi_encryption] - Last week Michael B. Steinbach, the assistant director of the FBI's Counterterrorism Division, gave testimony that puzzled many, claiming that the FBI was working with technology companies "to build technological solutions to prevent encryption above all else." This is just another in a long line of statements by law enforcement about the need to introduce a government "front door" into encryption standards. As I've mentioned in previous posts, cryptography experts argue that ["front doors" in encryption are very hard or impossible to get right][encryption_backdoors], and generally speaking, [are not a good idea][cloudflare_golden_key]. We're still facing the consequences of [weakened][freak] [encryption][weakdh], or "[export-grade encryption][export_encryption]," that was introduced in the last episode of the "crypto wars" in the '90s.
```
Comment (catlook) [inline _posts/2015-06-15-this-week-in-infosec-jun-15.md]: ... and, generally speaking, ...

File: _posts/2015-06-15-this-week-in-infosec-jun-15.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+[ruby_regex]: http://homakov.blogspot.ru/2012/05/saferweb-injects-in-various-ruby.html
+[ruby_regex2]: http://guides.rubyonrails.org/security.html#regular-expressions
+[bson2]: https://rubygems.org/gems/bson/versions/3.0.4
+
+[memex]: http://opencatalog.darpa.mil/MEMEX.html
+[memex2]: http://www.scientificamerican.com/article/human-traffickers-caught-on-hidden-internet/
+[memex3]: https://github.com/TeamHG-Memex/Formasaurus
+[memex4]: https://github.com/scrapinghub/splash
+
+[whatiscode]: http://www.bloomberg.com/graphics/2015-paul-ford-what-is-code/
```
Comment (catlook) [inline _posts/2015-06-15-this-week-in-infosec-jun-15.md]: [Proceed apace!!]

**Comment by cneill** [comment]
Thanks, Cat! Let me know if y'all need anything else from me.

**Pull request merged**